build/templates/report-designer/index.html

```diff
@@ -22,20 +22,25 @@
             }
         }
     </script>
-    <script type="text/javascript" src="../scripts/common/ej2-base.min.js"></script>
-    <script type="text/javascript" src="../scripts/common/ej2-data.min.js"></script>
-    <script type="text/javascript" src="../scripts/common/ej2-pdf-export.min.js"></script>
-    <script type="text/javascript" src="../scripts/common/ej2-svg-base.min.js"></script>
-    <script type="text/javascript" src="../scripts/data-visualization/ej2-circulargauge.min.js"></script>
-    <script type="text/javascript" src="../scripts/data-visualization/ej2-lineargauge.min.js"></script>
-    <script type="text/javascript" src="../scripts/data-visualization/ej2-maps.min.js"></script>
-
+    <script>
+        (function (w, d, s, l, i) {
+            w[l] = w[l] || []; w[l].push({
+                'gtm.start':
+                    new Date().getTime(), event: 'gtm.js'
+            }); var f = d.getElementsByTagName(s)[0],
+                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
+                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
+        })(window, document, 'script', 'dataLayer', 'GTM-K73VBT8');</script>
```

Comment:
@RamyaManickam1927  please properly remove the tag manager `gtm` scripts in all the places

Reply:
Done